src/_common/community/activity-item/AppCommunityActivityItem.vue

```diff
@@ -17,194 +17,179 @@ import { CommunityCompetitionModel } from '../competition/competition.model';
-	readonly PostFeature = CommunityActivityItemType.PostFeature;
-	readonly PostUnfeature = CommunityActivityItemType.PostUnfeature;
-	readonly PostMove = CommunityActivityItemType.PostMove;
-	readonly PostEject = CommunityActivityItemType.PostEject;
-
-	readonly ModInvite = CommunityActivityItemType.ModInvite;
-	readonly ModAccept = CommunityActivityItemType.ModAccept;
-	readonly ModRemove = CommunityActivityItemType.ModRemove;
-
-	readonly BlockUser = CommunityActivityItemType.BlockUser;
-
-	readonly EditDescription = CommunityActivityItemType.EditDescription;
-	readonly EditThumbnail = CommunityActivityItemType.EditThumbnail;
-	readonly EditHeader = CommunityActivityItemType.EditHeader;
-	readonly EditDetails = CommunityActivityItemType.EditDetails;
-	readonly EditHeaderRemove = CommunityActivityItemType.EditHeaderRemove;
-
-	readonly ChannelAdd = CommunityActivityItemType.ChannelAdd;
-	readonly ChannelRemove = CommunityActivityItemType.ChannelRemove;
-	readonly ChannelEdit = CommunityActivityItemType.ChannelEdit;
-	readonly ChannelRename = CommunityActivityItemType.ChannelRename;
-
-	readonly GameLink = CommunityActivityItemType.GameLink;
-	readonly GameUnlink = CommunityActivityItemType.GameUnlink;
-
-	readonly CompetitionEditSettings = CommunityActivityItemType.CompetitionEditSettings;
-	readonly CompetitionEditVoting = CommunityActivityItemType.CompetitionEditVoting;
-	readonly CompetitionVotingSetActive = CommunityActivityItemType.CompetitionVotingSetActive;
-	readonly CompetitionEntryRemove = CommunityActivityItemType.CompetitionEntryRemove;
-	readonly CompetitionEntryUnremove = CommunityActivityItemType.CompetitionEntryUnremove;
-	readonly CompetitionEntryGiveAward = CommunityActivityItemType.CompetitionEntryGiveAward;
-
-	readonly FiresideStart = CommunityActivityItemType.FiresideStart;
-	readonly FiresideStartDraft = CommunityActivityItemType.FiresideStartDraft;
-	readonly FiresidePublish = CommunityActivityItemType.FiresidePublish;
-	readonly FiresideExtinguish = CommunityActivityItemType.FiresideExtinguish;
-	readonly FiresideFeature = CommunityActivityItemType.FiresideFeature;
-	readonly FiresideUnfeature = CommunityActivityItemType.FiresideUnfeature;
-	readonly FiresideEject = CommunityActivityItemType.FiresideEject;
-
-	get loggedOn() {
-		return formatDate(this.item.added_on, 'medium');
-	}
-
-	get shouldShowIcon() {
-		return !!this.icon && this.showIcon;
-	}
-
-	get icon() {
-		return this.item.getTypeIcon()?.icon;
-	}
-
-	get color() {
-		return this.item.getTypeIcon()?.color;
-	}
-
-	get actionIsFiresidePost() {
-		return this.item.action_resource instanceof FiresidePostModel;
-	}
-
-	get actionIsUser() {
-		return this.item.action_resource instanceof UserModel;
-	}
-
-	get isToday() {
-		return (
-			formatDate(this.item.added_on, 'mediumDate') === formatDate(Date.now(), 'mediumDate')
-		);
+	showIcon: {
+		type: Boolean,
+		required: true,
+	},
+});
+
+const CommunityCreated = CommunityActivityItemType.CommunityCreated;
```

Comment:
Instead, just use the enums directly in the file, no need for these intermediate consts anymore